op-node/node/node.go

```diff
@@ -568,6 +568,33 @@ func (n *OpNode) Start(ctx context.Context) error {
 		return err
 	}
 	log.Info("Rollup node started")
+
+	// If n.cfg.Driver.SequencerUseFinalized is true, sequencer cannot retrieve non-finalized L1 blocks.
+	// OpNode periodically fetches the latest safe and finalized L1 blocks every epoch (13 minutes),
```

Comment:
That should be 6.4 minutes I think. An epoch is 32*12 seconds, which is the default for the `L1EpochPollIntervalFlag`